Remove unused argument "destId" in addSendRpcBackward

xush6528 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#31207 Remove unused argument "destId" in addSendRpcBackward**
* #31206 Remove the second copy on calling dist_autograd_context._known_worker_ids()

Cleanup after #30914.

In #30914, `autogradContext->addKnownWorkerId(dst);` was moved out of `addSendRpcBackward()`.

So `addSendRpcBackward()` does not need `dstId` as it's argument anymore.

Differential Revision: [D5742365](https://our.internmc.facebook.com/intern/diff/D5742365/)

xush6528 committed:
Remove unused argument "destId" in addSendRpcBackward

Cleanup after #30914.

In #30914, `autogradContext->addKnownWorkerId(dst);` was moved out of `addSendRpcBackward()`.

So `addSendRpcBackward()` does not need `dstId` as it's argument anymore.

Differential Revision: [D5742365](https://our.internmc.facebook.com/intern/diff/D5742365/)

[ghstack-poisoned]

rohan-varma reviewed (approved):
Thanks!

kostmo commented:
## CircleCI build failures summary
As of commit cc1339f:
* **1/1** failures introduced in this PR



## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=cc1339f55f30e5e62bcf08e3e65e7277d3bc82c2).

### 1 new failure recognized by patterns
The following build failures don't appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3930006) pytorch_linux_xenial_py3_6_gcc5_4_test (1/1)
**Step:** "Test" ([details](https://dr.pytorch.org/build-details.html?build_id=73556068))
```
Dec 12 21:32:02 Ran 48 tests in 58.116s 
Dec 12 21:32:02  
Dec 12 21:32:02 FAILED (failures=2, skipped=2) 
Dec 12 21:32:02  
Dec 12 21:32:02 Generating XML reports... 
Dec 12 21:32:02 Traceback (most recent call last): 
Dec 12 21:32:02   File "test/run_test.py", line 456, in <module> 
Dec 12 21:32:02     main() 
Dec 12 21:32:02   File "test/run_test.py", line 449, in main 
Dec 12 21:32:02     raise RuntimeError(message) 
Dec 12 21:32:02 RuntimeError: test_dist_autograd_spawn failed! 
Dec 12 21:32:02 + cleanup 
Dec 12 21:32:02 + retcode=1 
Dec 12 21:32:02 + set +x 
Dec 12 21:32:02 =================== sccache compilation log =================== 
Dec 12 21:32:02 ERROR:sccache::server: Compilation failed: Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "/tmp/torch_extensions/test_compilation_error_formatting/main.cpp: In function \'int main()\':\n/tmp/torch_extensions/test_compilation_error_formatting/main.cpp:2:23: error: expected \';\' before \'}\' token\n int main() { return 0 }\n                       ^\n" } 
Dec 12 21:32:02  
Dec 12 21:32:02 =========== If your build fails, please take a look at the log above for possible reasons =========== 
Dec 12 21:32:02 Compile requests                115 
Dec 12 21:32:02 Compile requests executed        25 
Dec 12 21:32:02 Cache hits                       18 
```


---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

xush6528 committed:
Update on "Remove unused argument "destId" in addSendRpcBackward"


Cleanup after #30914.

In #30914, `autogradContext->addKnownWorkerId(dst);` was moved out of `addSendRpcBackward()`.

So `addSendRpcBackward()` does not need `dstId` as it's argument anymore.

Differential Revision: [D5742365](https://our.internmc.facebook.com/intern/diff/D5742365/)

[ghstack-poisoned]
